### DISCUSSION TITLE
qbittorrent

### DISCUSSION THREAD

**Discussion by PerkelSimon** [userdocs/qbittorrent-nox-static#184] [General]
@userdocs  eh, it is possible to create a package from your qbittorrent for open wrt firmware?

**Comment by userdocs** [comment]
As in me create a package or you?

Because if it's me, i'm not going to do it. It's outside the scope of the project.

**Reply by PerkelSimon** [reply]
no, maybe just to have some infos to helps.

**Reply by userdocs** [reply]
why do you need to package it? it runs fine in openwrt as a binary.

Create user for qbittorrent - https://openwrt.org/docs/guide-user/additional-software/create-new-users

install with generic install instructions - https://github.com/userdocs/qbittorrent-nox-static?tab=readme-ov-file#x86

That's all there is too it.
